packages/p/pinta/package.yml

```diff
@@ -1,32 +1,31 @@
 name       : pinta
-version    : 1.7.1
-homepage   : https://pinta-project.com/pintaproject/pinta/
-release    : 13
+version    : 2.1.1
+homepage   : https://www.pinta-project.com/
+release    : 14
 source     :
-    - https://github.com/PintaProject/Pinta/releases/download/1.7.1/pinta-1.7.1.tar.gz : cdbbbfe241b8fe5f3a1d0b16e73115125e264a9c94d25fce9e2fcf43429efab9
+    - https://github.com/PintaProject/Pinta/releases/download/2.1.1/pinta-2.1.1.tar.gz : 32cedc983e9c50e1bc15a85b0fc909f52e97c7a0f5cca3a2c47b711a717df9eb
 license    : MIT
 component  : multimedia.graphics
 summary    : Pinta Painting Made Simple
 description: |
     Pinta is a free, open source drawing/editing program modeled after Paint.NET. Its goal is to provide users with a simple yet powerful way to draw and manipulate images on Linux, Mac, Windows, and *BSD.
+libsplit   : no
+networking : yes
 builddeps  :
-    - pkgconfig(gtk-sharp-2.0)
-    - pkgconfig(mono)
-    - pkgconfig(mono-addins)
-    - pkgconfig(yelp-xsl)
-    - itstool
+    - pkgconfig(gtk+-3.0)
+    - autoconf-archive
+    - dotnet-sdk
+    - gettext-devel
     - intltool
-    - yelp-devel
-    - yelp-tools
-rundeps    :
-    - mono
-    - mono-addins
-    - gtk-sharp
-patterns   :
-    - /usr/lib64/pkgconfig
 setup      : |
-    %configure
+    %patch -p1 -i $pkgfiles/0001-use-net8.patch
+    %reconfigure
+    sed -i '/^PINTA_BUILD_OPTS/ s/$/ -maxcpucount:1/' Makefile.in
```

Comment:
This line probably isn't doing anything since the makefile is generated from the makefile.in during the %reconfigure

Reply:
It's needed. It doesn't work before autoreconf since that rebuilds the Makefile.in

```sh
+ make -j24
 cd . && /bin/sh ./config.status Makefile 
config.status: creating Makefile
cd . && env TargetFramework=net8.0 /usr/bin/dotnet build Pinta.sln --configuration Release -p:BuildTranslations=true -maxcpucount:1
Making all in xdg
```
I'll put it before the `%make` so it doesn't look like an oversight